docs/html/reference/pip_install.rst

```diff
@@ -384,16 +384,14 @@ where ``setup.py`` is not in the root of project, the "subdirectory" component
 is used. The value of the "subdirectory" component should be a path starting
 from the root of the project to where ``setup.py`` is located.
 
-So if your repository layout is:
-
-    - pkg_dir/
-
-      - setup.py  # setup.py for package ``pkg``
-      - some_module.py
-    - other_dir/
-
-      - some_file
-    - some_other_file
+So if your repository layout is::
+
+   pkg_dir
+   ├── setup.py  # setup.py for package "pkg"
+   └── some_module.py
+   other_dir
+   └── some_file
+   some_other_file
 
 You'll need to use ``pip install -e "vcs+protocol://repo_url/#egg=pkg&subdirectory=pkg_dir"``.
```

Comment:
We should also address this line not rendering properly when the documentation is opened in chrome

Reply:
That can be done by using a code block then, but don't you think that's an upstream (either sphinx or chromium) issue?